### PR TITLE
Tmhmm.py

### PR DESCRIPTION
**Pull request by CalounovaT**
Implemented [tmhmm.py](https://github.com/dansondergaard/tmhmm.py) instead of [TMHMM](http://www.cbs.dtu.dk/services/TMHMM/) in rule tmhmm_parallel as mentioned in #17.
The output is slightly different and contains only number of expected helices and short topology string.

Also it seems like it needs to be installed via [pip](https://pypi.org/project/tmhmm.py/) since [conda](https://anaconda.org/dansondergaard/tmhmm.py) version is not maintained and did not behave as expected.

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
Thanks! Please also update the README to reflect the new way of installing the tmhmm.py dependency.

**Comment by CalounovaT** [comment]
> Thanks! Please also update the README to reflect the new way of installing the tmhmm.py dependency.

README updated.

File: README.md [hunk 1]
```diff
@@ -53,8 +53,12 @@ conda install snakemake fastqc trimmomatic trinity spades transdecoder biopython
          python setup.py install
         ~~~~
         (make sure the conda python is used, or use the full path to python from your conda installation)
+5. Instal [tmhmm.py](https://github.com/dansondergaard/tmhmm.py) via pip:
```
Comment (tomas-pluskal) [inline README.md:56]: instal -> install

**Comment by tomas-pluskal** [comment]
Thank you! 👍

**Pull request merged**